awesomplete.js

```diff
@@ -39,10 +39,12 @@ var _ = function (input, o) {
 
 	// Create necessary elements
 
-	this.container = $.create("div", {
-		className: "awesomplete",
-		around: input
-	});
+	this.container = input.parentNode.className.search(/awesomplete-wrapper/) >= 0 ?
+		input.parentNode :
```

Comment:
No need for a ternary here. You can just do something like `$(".awesomplete-wrapper", parent) || $.create(...)`